upgrade.yml

```diff
@@ -669,4 +669,18 @@
         endpoints:
           db: "{{ undercloud_floating_ip }}:3307"
 
-- include: site.yml
+- hosts: controller
+  gather_facts: no
+  serial: 1
+  tasks:
+  - name: run neutron-reset on controllers
+    shell: /usr/local/bin/neutron-reset
```

Comment:
can we get an addition play after the controller one targeting compute which does `neutron-reset` non-serialized ?